main.py

```diff
@@ -0,0 +1,318 @@
+import argparse
+import os
+
+import pandas as pd
+import math
+import numpy as np
+from census import Census
+from us import states
+from pygris import tracts
+import matplotlib.pyplot as plt
+
+from config import FIELDS, REGION, WEIGHTS
+from utils import logging
+
+# Get an API key here: https://api.census.gov/data/key_signup.html
+CENSUS_API_KEY = os.getenv("CENSUS_API_KEY")
+c = Census(CENSUS_API_KEY)
+
+
+def load(year):
+    # we have to match the sorting of the original work
+    # so this query we will merge to preserve the sorting we need
+    sort_query = c.acs5.state_county_tract(
+        fields=["NAME", "B01001_001E"],
+        state_fips=states.TX.fips,
+        county_fips="*",
+        tract="*",
+        year=year,
+    )
+    tx_census = c.acs5.state_county_tract(
+        fields=FIELDS,
+        state_fips=states.TX.fips,
+        county_fips="*",
+        tract="*",
+        year=year,
+    )
+    df = pd.DataFrame(tx_census)
+    sort_query = pd.DataFrame(sort_query)
+    sort_query = sort_query[sort_query["county"].isin(REGION)]
+    sort_query = sort_query[["NAME"]]
+    df = df[df["county"].isin(REGION)]
+    df = sort_query.merge(df, on="NAME", how="left")
+    return df
+
+
+def transform(census_data):
```

Comment:
this chunk of code largely follows [process_data.R](https://github.com/cityofaustin/vulnerability_weighting_map/blob/main/process_data.R)